docs/source/api/utils.rst

```diff
@@ -0,0 +1,7 @@
+utils
```

Comment:
I just noticed by going through this module's API docs that there's a `# noqa` that's rendered. Maybe we can avoid this.

It's coming from the docstring in `utils.get_image`. So maybe we can replace this line:
```
    References
    ----------
    [1] https://allensdk.readthedocs.io/en/latest/allensdk.api.queries.image_download_api.html#allensdk.api.queries.image_download_api.ImageDownloadApi  # noqa
```
by this:
```
    References
    ----------
    [1] `AllenSDK API: ImageDownloadApi <https://allensdk.readthedocs.io/
    en/latest/allensdk.api.queries.image_download_api.html#allensdk.api.
    queries.image_download_api.ImageDownloadApi>`_
```